keithleyClient/keithleyInterface.py

```diff
@@ -68,7 +68,7 @@ def read(self,minLength=0):
             time.sleep(self.readSleepTime)
             i+=1
         ts =time.time()
-        maxTime = 20
+        maxTime = 300
```

Comment:
I agree we can increase the time to 300seconds. Maybe we can move it into a config file variable. I ll check that.
We made an investigation some days ago and we found out that we can fix this issue by fixing the measurement range to 100muA.
We are testing it at the moment